node/mint.go

```diff
@@ -0,0 +1,45 @@
+package node
+
+import "github.com/pegnet/pegnetd/fat/fat2"
+
+// Special structure for the mint amount
+type MintSupply struct {
+	Ticker fat2.PTicker // token
+	Amount uint64       // amount of token to mint
+}
+
+var (
+	MintTotalSupplyMap = []MintSupply{
+		{fat2.PTickerPEG, 334509613},
+		{fat2.PTickerUSD, 3184409},
+		{fat2.PTickerKRW, 118},
+		{fat2.PTickerXAU, 1},
+		{fat2.PTickerXAG, 599},
+		{fat2.PTickerXBT, 2},
+		{fat2.PTickerETH, 5476},
+		{fat2.PTickerLTC, 2004},
```

Comment:
Some of these, like LTC and BNB don't match the amounts in https://discord.com/channels/@me/803656757121515540/818940522713317417

Reply:
If you open google doc in that chat, there is final tab which was calculated from both pegnetd candidates as upper limit.